app/grandchallenge/challenges/models.py

```diff
@@ -558,9 +559,67 @@ def remove_admin(self, user):
 
     @property
     def accepting_submissions(self):
-        return True in (
-            phase.open_for_submissions for phase in self.phase_set.all()
-        )
+        phase_details = {
+            phase: phase.open_for_submissions for phase in self.phase_set.all()
+        }
+        now = timezone.now()
+        if True in phase_details.values():
+            status = True
+            if sum(phase_details.values()) > 1:
+                # if there are multiple open phases it is unclear which
+                # status to print, so stay vague
+                detail = ["Accepting submissions"]
+            else:
+                detail = [
+                    phase.submission_status_string
+                    for phase, bool in phase_details.items()
+                    if bool
+                ]
+        else:
+            status_strings = [
+                phase.submission_status_string
+                for phase in phase_details.keys()
+            ]
+            completed_phases = [
+                string for string in status_strings if "completed" in string
```

Comment:
Checking for substrings in strings is going to be fragile. I think a better way would be to add a property to the phase that is an enum to handle all of these cases, eg all the ones found in `submission_status_string`:

- Closed
- Opening Soon
- Open
- Completed